sdk/search/Azure.Search.Documents/src/Generated/Models/CustomEntity.cs

```diff
@@ -0,0 +1,85 @@
+        public string Description { get; set; }
+        /// <summary> This field can be used as a passthrough for custom metadata about the matched text(s). The value of this field will appear with every match of its entity in the skill output. </summary>
+        public string Type { get; set; }
+        /// <summary> This field can be used as a passthrough for custom metadata about the matched text(s). The value of this field will appear with every match of its entity in the skill output. </summary>
+        public string Subtype { get; set; }
+        /// <summary> This field can be used as a passthrough for custom metadata about the matched text(s). The value of this field will appear with every match of its entity in the skill output. </summary>
+        public string Id { get; set; }
+        /// <summary> Defaults to false. Boolean value denoting whether comparisons with the entity name should be sensitive to character casing. Sample case insensitive matches of &quot;Microsoft&quot; could be: microsoft, microSoft, MICROSOFT. </summary>
+        public bool? CaseSensitive { get; set; }
+        /// <summary> Defaults to false. Boolean value denoting whether comparisons with the entity name should be sensitive to accent. </summary>
+        public bool? AccentSensitive { get; set; }
+        /// <summary> Defaults to 0. Maximum value of 5. Denotes the acceptable number of divergent characters that would still constitute a match with the entity name. The smallest possible fuzziness for any given match is returned. For instance, if the edit distance is set to 3, &quot;Windows10&quot; would still match &quot;Windows&quot;, &quot;Windows10&quot; and &quot;Windows 7&quot;. When case sensitivity is set to false, case differences do NOT count towards fuzziness tolerance, but otherwise do. </summary>
+        public int? FuzzyEditDistance { get; set; }
+        /// <summary> Changes the default case sensitivity value for this entity. It be used to change the default value of all aliases caseSensitive values. </summary>
+        public bool? DefaultCaseSensitive { get; set; }
+        /// <summary> Changes the default accent sensitivity value for this entity. It be used to change the default value of all aliases accentSensitive values. </summary>
+        public bool? DefaultAccentSensitive { get; set; }
+        /// <summary> Changes the default fuzzy edit distance value for this entity. It can be used to change the default value of all aliases fuzzyEditDistance values. </summary>
+        public int? DefaultFuzzyEditDistance { get; set; }
+        /// <summary> An array of complex objects that can be used to specify alternative spellings or synonyms to the root entity name. </summary>
+        public IList<CustomEntityAlias> Aliases { get; set; }
```

Comment:
I know we've talked about it, but we haven't made this change yet, have we? I.e. collection properties shouldn't be settable.

Reply:
Per internal discussion, these shouldn't be settable since they are collection properties. May be a bug in the generator. /cc @ShivangiReja @pakrym

Reply:
What should be the resolution?
An update to the generator followed by code regen?

Reply:
Pretty sure this property is nullable so we are forced to make it settable.

Reply:
In Search we already had that issue and resolved it without making them settable. Previously we used the NullAsEmpty attribute property (or something like that), but I don't remember where we landed finally off hand. Pretty sure it wasn't settable collection properties. @Mohit-Chakraborty , I recommend looking at existing models with collection properties and see what we're doing.